.eslintrc.js

```diff
@@ -17,7 +17,7 @@ module.exports = {
     "react/jsx-uses-vars": 2,
     "react/no-array-index-key": 1,
     "react/destructuring-assignment": 1,
-    "import/extensions": ['error', 'always', { ignorePackages: true }],
+    "import/extensions": "ignore",
```

Comment:
nice work. I did this too to get rid of tyler's dreadful es lint errors for Material UI